install/migrations/update_10.0.x_to_11.0.0/configs.php

```diff
@@ -44,10 +44,17 @@
     'set_followup_tech' => '0',
     'set_solution_tech' => '0',
     'is_demo_dashboards' => '0',
+    'planned_task_state' => Planning::TODO,
```

Comment:
It is preferable to use only raw values in the migration code. This prevents the migration behaviour to change if the corresponding constant value changes.

```suggestion
    'planned_task_state' => '1',
```